docs/14-batch-shipyard-configuration-jobs.md

```diff
@@ -701,9 +701,12 @@ configuration. `id` should not be specified in conjunction with the
 `task_factory` property as `id`s will be automatically generated. A
 task `id` may not exceed 64 characters in length.
 * (required if using a Docker image) `docker_image` is the Docker image to
-use for this task
+use for this task.
 * (required if using a Singularity image) `singularity_image` is the
-Singularity image to use for this task
+Singularity image to use for this task. Due to Singularity limitations, if the
+image specified at a certain URI changes, the image will need to be pulled
+again from the registry the next time that the image is used in a task which
+will lead to increased latency to begin task execution.
```

Comment:
begin task execution, and lead to potential inconsistencies between task executions.